health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/attendance_inbox.js

```
@@ -0,0 +1,26 @@
+  const { t } = useTranslation();
+  const location = useLocation();
+
+  return (
+    <React.Fragment>
+      <Header styles={{ fontSize: "32px" }}>
+        {t("HCM_AM_ATTENDANCE_INBOX")}
+        {location?.state?.count ? <span className="inbox-count">{location?.state?.count}</span> : null}
+      </Header>
+      <div className="inbox-search-wrapper">
+        {/* <InboxSearchComposer configs={configs}></InboxSearchComposer>*/}
+        <CustomInboxSearchComposer></CustomInboxSearchComposer>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)